### PR TITLE
Designers in Spanish

### PR DESCRIPTION
**Pull request by fafranco82**

### CODE REVIEW

File: translations/es/pack/SoD.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    {
+        "code": "10020",
+        "flavor": "\"You mistake patience for forbearance. I have worked at the downfall of Tywin Lannister since the day they told me of Elia and her children.\" <cite>Doran Martell</cite>",
+        "name": "Patience",
+        "text": "Character you control only.\n<b>Challenges Action:</b> Return attached character to its owner's hand.",
+        "traits": "Skill."
+    },
+    {
+        "code": "10039",
+        "flavor": null,
```
Comment (ystros) [inline translations/es/pack/SoD.json]: This fails validation on the translation file but seems to be fine in the English source. Looking at the schema, `flavor` is allowed to be null or an empty string but the translation file schema requires it to be an empty string or missing.

Dropping this line should get rid of the validation error, the schema inconsistency should probably be addressed separately.

File: translations/es/pack/OR.json [hunk 2]
```diff
@@ -58,10 +58,10 @@
     },
     {
         "code": "06090",
-        "flavor": "\"Recuerdo perfectamente qué le dije. Pero el Castillo Negro no es Harrenhal, y la Guardia de la Noche no es el Consejo del Rey. Hay una herramienta para cada tarea, y una tarea para cada herramienta.\" <cite>Tyrion Lannister</cite>",
         "name": "Una tarea para cada herramienta",
         "text": "<b>Acción de Reto:</b> pon en juego desde tu mano un Personaje [lannister] que tenga una FUE impresa de 2 o menos.",
-        "traits": ""
+        "traits": "",
+        "flavor": "\"Recuerdo perfectamente qué le dije. Pero el Castillo Negro no es Harrenhal, y la Guardia de la Noche no es el Consejo del Rey. Hay una herramienta para cada tarea, y una tarea para cada herramienta.\" <cite>Tyrion Lannister</cite>"
```
Comment (ystros) [inline translations/es/pack/OR.json]: Moving the `flavor` property outside of alphabetical order generates an error when running `validate.py`. Don't see any other changes to the text, so just moving it back should resolve the issue.

File: translations/es/pack/TFoA.json [hunk 3]
```diff
@@ -102,10 +102,10 @@
     },
     {
         "code": "06057",
-        "flavor": "En cierta ocasión, su padre le había dicho que Walder Frey era el único señor de los Siete Reinos capaz de sacarse un ejército de los calzones.",
         "name": "Hidalgo de los Frey",
         "text": "<b>Reacción:</b> después de que inicies un reto, el Hidalgo de los Frey recibe un +1 a su FUE hasta el final de la fase.",
-        "traits": "Casa Frey. Lord."
+        "traits": "Casa Frey. Lord.",
+        "flavor": "En cierta ocasión, su padre le había dicho que Walder Frey era el único señor de los Siete Reinos capaz de sacarse un ejército de los calzones."
```
Comment (ystros) [inline translations/es/pack/TFoA.json]: Validation error from ordering change to `flavor` property.

File: translations/es/pack/TKP.json [hunk 4]
```diff
@@ -97,10 +98,10 @@
     },
     {
         "code": "02056",
-        "flavor": "Any knight can make a knight...",
         "name": "El Sendahueso",
         "text": "<b>Reacción:</b> después de que pierdas un reto, coloca 1 ficha de Venganza sobre El Sendahueso.\n<b>Acción de Dominación:</b> arrodilla El Sendahueso y descarta 6 fichas de Venganza que tenga encima para ganar 3 de Poder para tu facción.",
-        "traits": "Dorne."
+        "traits": "Dorne.",
+        "flavor": "Any knight can make a knight..."
```
Comment (ystros) [inline translations/es/pack/TKP.json]: Validation error from ordering change to `flavor` property.

File: translations/es/pack/TRW.json [hunk 5]
```diff
@@ -70,10 +71,10 @@
     },
     {
         "code": "06072",
-        "flavor": "El único trabajo apto para un hombre del hierro era la guerra.",
         "name": "Saqueo",
         "text": "<b>Acción:</b> arrodilla tu carta de Facción para elegir un adversario. Ganas 1 de Oro por cada Lugar y Accesorio que haya en la pila de descartes de ese jugador.",
-        "traits": ""
+        "traits": "",
+        "flavor": "El único trabajo apto para un hombre del hierro era la guerra."
```
Comment (ystros) [inline translations/es/pack/TRW.json]: Validation error from ordering change to `flavor` property.

**Comment by fafranco82** [comment]
Now must be fixed... at least the formatting issue.

**Comment by ystros** [comment]
Looks good to go assuming @Alsciende is fine w/ the schema change.

**Comment by Alsciende** [comment]
I don't think there's an impact on the code. I'll merge this and see what happens :)

**Pull request merged**